plugins/processors/multiplier/multiplier.go

```diff
@@ -37,6 +37,9 @@ func (multiplier *Multiplier) Description() string {
 	return "Multiply metrics values on some multiply factor"
 }
 
+var totalMhz_average float64 = 0
+var effectivecpu_average float64 = 0
```

Comment:
@t-arjkas, initial look at this, I'm wondering if this is goroutine safe. May be move it into the function.